Client/Frontend/Browser/TabManager.swift

```diff
@@ -63,9 +63,10 @@ class TabManager: NSObject {
 
     fileprivate let navDelegate: TabManagerNavDelegate
 
-    public static func makeWebViewConfig(isPrivate: Bool, blockPopups: Bool) -> WKWebViewConfiguration {
+    public static func makeWebViewConfig(isPrivate: Bool, prefs: Prefs?) -> WKWebViewConfiguration {
         let configuration = WKWebViewConfiguration()
         configuration.processPool = WKProcessPool()
+        let blockPopups = prefs?.boolForKey("blockPopups") ?? true
```

Comment:
nit: Would be nice to move block popups to prefskey
```
public static let KeyBlockPopups = "blockPopups"
let blockPopups = prefs?.boolForKey(PrefsKeys.KeyBlockPopups) ?? true
```

Reply:
will do!